.github/workflows/ci.yml

```diff
@@ -93,7 +93,11 @@ jobs:
                 . ./ci-support-v0
 
                 if test "$DOWNSTREAM_PROJECT" = "mirgecom"; then
-                    git clone "https://github.com/illinois-ceesd/$DOWNSTREAM_PROJECT.git"
+                    if [[ "$GITHUB_HEAD_REF" = "nodal-reduction-device-scalar" ]]; then
+                        git clone "https://github.com/majosm/$DOWNSTREAM_PROJECT.git" -b "nodal-reduction-device-scalar"
+                    else
+                        git clone "https://github.com/illinois-ceesd/$DOWNSTREAM_PROJECT.git"
+                    fi
```

Comment:
We'll need to drop this before merging

Reply:
I _think_ this needs to stay for now, otherwise the mirgecom CI job will fail and I won't be able to merge. It can be removed again once illinois-ceesd/mirgecom#518 is also merged. It won't affect anything in main since the branch name isn't `nodal-reduction-device-scalar`.